File: code/library/src/main/java/org/fbme/lib/iec61499/ecc/ECC.java
package org.fbme.lib.iec61499.ecc;


import org.fbme.lib.common.Element;
import org.jetbrains.annotations.NotNull;

import java.util.List;

public interface ECC extends Element {

    @NotNull List<StateDeclaration> getStates();

    @NotNull List<StateTransition> getTransitions();
}

File: code/library/src/main/kotlin/org/fbme/lib/iec61499/ecc/ECC.kt
package org.fbme.lib.iec61499.ecc

import org.fbme.lib.common.Element
import org.fbme.lib.iec61499.declarations.BasicFBTypeDeclaration

interface ECC : Element {
    val states: MutableList<StateDeclaration>
    val transitions: MutableList<StateTransition>
    override val container: BasicFBTypeDeclaration
}
